pkg/config/env.go

```diff
@@ -42,6 +42,8 @@ var Config = struct {
 	// EvalOnlyMode - will only expose the evaluation related endpoints.
 	// This field will be derived from DBDriver
 	EvalOnlyMode bool `env:"FLAGR_EVAL_ONLY_MODE" envDefault:"false"`
+	// additional values to put in EntityContext when evaluating segments
```

Comment:
the comment can have more information like how the keys can be transitioned into `@` prefix, better if it comes with a simple example.